src/zsl/utils/security_helper.py

```diff
@@ -50,8 +50,14 @@ def compute_token(random_token, config):
     :rtype: str
     """
     secure_token = config[TOKEN_SERVICE_SECURITY_CONFIG]
+    msg_to_hash = random_token + secure_token
+
     sha1hash = hashlib.sha1()
-    sha1hash.update(random_token + secure_token)
+    try:
+        sha1hash.update(msg_to_hash.encode('utf-8'))
+    except TypeError:
+        sha1hash.update(msg_to_hash)
```

Comment:
Do not use exceptions in this case. Just create two methods:
```
if py_ver < 3:
  def meyhod_to_obtain_xy():
    ...
else:
  def meyhod_to_obtain_xy():
    ...
```

and then use the method.